src/webui/www/private/scripts/mocha-init.js

```diff
@@ -163,6 +162,33 @@ initializeWindows = function() {
         }
     };
 
+    shareRatioFN = function() {
+        console.log("Limit share ratio...");
+        var hashes = torrentsTable.selectedRowsIds();
+        if (hashes.length) {
+            var origValues = "";
+            if (hashes.length === 1) {
+                var hash = hashes[0];
+                var row = torrentsTable.rows[hash].full_data;
+                console.log(row);
```

Comment:
Is this intended?

Reply:
Don't commit code in the early hours of the morning. Removing this.